modules/moa-accessing-your-cluster.adoc

```diff
@@ -8,12 +8,12 @@
 
 To log in to your cluster, you must configure an Identity Provider (IDP).
 
-This guide uses GitHub as an example IDP. 
+This procedure uses GitHub as an example IDP. 
 
 .Procedure
 
 . Add an IDP.
-.. Enter the following command to create an IDP backed by GitHub. Follow the prompts from the output to access your [GitHub developer settings](https://github.com/settings/developers) and configure a new OAuth application.
+.. Enter the following command to create an IDP backed by GitHub. Follow the prompts from the output to access your link:https://github.com/settings/developers[GitHub developer settings] and configure a new OAuth application.
```

Comment:
In the first sentence, I might move the goal to the front of the sentence. Also, this seems like two separate steps. I suggest moving "Follow the prompts" after the terminal command so it's more in sequence. I get that the output is long, though, so not sure that it will work but you could try putting the example output after the "Follow the prompts" step.

So maybe something like:

```
.. To create an IDP backed by GitHub, enter the following command: 
+
[source, terminal]
----
$ moactl create idp --cluster rh-moa-test-cluster1 --type github                                                                                               
----
+
.. Follow the prompts from the output to access your link:https://github.com/settings/developers[GitHub developer settings] and configure a new OAuth application.

+
.Example output
[source,terminal]
----
`...`
```

Reply:
If you do end up front-loading the goal here, you'll need to do it elsewhere for consistency. Not sure if that blows up the rest of your docs conventions, though, so just a suggestion.

Reply:
Great suggestion. I actually have to update this section to use the interactive flag as we did in the upstream. I'll also test out separating the initial command from the output with some info in between. https://github.com/openshift/moactl#add-an-idp